### DISCUSSION TITLE
Unable to sign in to partner.microsoft.com

### DISCUSSION THREAD

**Discussion by mauricelam** [microsoft/MicrosoftEdge-Extensions#133] [Q&A]
I am trying to publish an update to my extension but when I try to sign in to "Developer dashboard" (https://aka.ms/PartnerCenterLogin) I got the error 

```
Sign in required

Your session has expired. Please sign-in again to continue. Unfortunately any unsaved changes will be lost.
```

Clicking sign in brings me to the sign in page, but after signing in it still shows the same error message. Clicking sign in again brings me back to the same page with the same error after a few automatic redirects. I tried signing out and back in, and tried with a different browser, but still the same issue.

I also have trouble reaching out through regular support channels since I am not properly signed in. Anyone in this community would be able to help with this?

**Comment by ManikanthMSFT** [comment]
Hi @mauricelam, thank you for providing the details. We apologize for the inconvenience you’re experiencing with the Developer Dashboard.

Could you please confirm if this issue is still persisting or if it has been resolved? This will help us assist you more effectively.

Thank you for your patience and cooperation.

**Reply by mauricelam** [reply]
The issue has been resolved, though I don't know what made the difference. Thanks for following up.
